sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -842,6 +842,333 @@ class FileBasedDataSourceSuite extends QueryTest
       }
     }
   }
+
+  test("SPARK-31116: Select simple parquet with case insensitive and schema pruning enabled") {
```

Comment:
Hi, @kimtkyeom . 
Why did you add this test case? This does not fail at `master`. If there is no other reason, please remove this test case. Otherwise, this only causes confusion to reviews.